tools/upload_to_aws.py

```diff
@@ -49,6 +49,11 @@
 
     try:
         print subprocess.check_output(['icgc-storage-client','upload','--file', file_, '--object-id', object_id, '--md5', file_md5sum, '--force'])
```

Comment:
Can we add `--profile` option for the `icgc-storage-client` command? I know that it defaults to aws, but I think we should be explicit about it.